relay-server/src/actors/outcome.rs

```diff
@@ -427,6 +427,7 @@ mod processing {
                 counter(RelayCounters::Outcomes) += 1,
                 reason = message.reason.as_deref().unwrap_or(""),
                 outcome = message.tag_name(),
+                to = "kafka",
```

Comment:
Please also describe the new tag in the doc comment on the `RelayCounters::Outcomes` enum variant.